src/app/static/src/app/store/modelCalibrate/actions.ts

```diff
@@ -117,6 +121,22 @@ export const actions: ActionTree<ModelCalibrateState, RootState> & ModelCalibrat
         if (response) {
             commit(ModelCalibrateMutation.SetPlotData, response.data);
         }
+    },
+
+    async getComparisonPlot(context) {
+        const {commit, state} = context;
+        const calibrateId = state.calibrateId;
+        commit(ModelCalibrateMutation.ComparisonPlotStarted);
```

Comment:
I'm not sure how useful a `generating..` flag is - we'd generally use such things when we're polling, but this is just a single request.  Is the idea that we'd display a spinner instead of the plot while this is true? We could probably just rely on the absence of the plot data to tell us that (so we would want to reset that here).